tests/material_tests.cc

```diff
@@ -208,6 +209,28 @@ TEST_F(MaterialTest, ExtractInGrams) {
   EXPECT_DOUBLE_EQ(test_size_ - kg_to_rem, default_mat_->quantity());
 }
 
+TEST_F(MaterialTest, DecayResBuf) {
+  // prequeries
+  cyclus::toolkit::MatQuery orig(tracked_mat_);
+  double u235_qty = orig.mass(u235_);
+  double pb208_qty = orig.mass(pb208_);
+  double am241_qty = orig.mass(am241_);
+  double orig_mass = tracked_mat_->quantity();
+
+  cyclus::toolkit::ResBuf<cyclus::Material> res_buf;
+  res_buf.Push(tracked_mat_);
+  res_buf.Decay(100);
+  cyclus::Material::Ptr pop_mat = res_buf.Pop();
+
+  // postquery
+  cyclus::toolkit::MatQuery mq(pop_mat);
+
+  // postchecks
```

Comment:
I was lazy and just copied the test done for Decay for a single material, because those were in the original `tracked_mat_`.  I guess I could add to that, though.

Reply:
I added Sr-89 with a 50.57 day 1/2 life